projects/plugins/jetpack/extensions/blocks/social-previews/block.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This is more of a plugin than a block, I think we could leave it out.

Reply:
This is right, `social-previews` registers only a plugin (adding stuff to sidebar and the pre-publish panel), there's no block at all. But removing `payments-intro/block.json` was a mistake.